docs/python-examples/fixtures/ocs.py

```diff
@@ -0,0 +1,71 @@
+import random
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_random_name(obj_type):
+    """
+    Random
+    """
+    rand = random.randint(1000, 9999)
+    return f"{obj_type}_{rand}"
+
+
+def create_pvcs(count, storage_class):
+    """
+    Helper function for creating PVCs
+
+    Args:
+        count (int): number of PVC to create
+        storage_class (StorageClass): storage class object reference
+
+    Returns:
+        list: PVCs objects
+    """
+    _pvcs = []
+    for number in range(count):
+        pvc_name = get_random_name('pvc')
+        _pvcs.append(PVC(pvc_name, storage_class))
+    return _pvcs
+
+
+class PVC:
```

Comment:
Why do we need these classes if we can use helpers.py create resource functions?

Reply:
It's just and dummy class which represent the resource class for the examples of fixtures. Those are just short dummy examples and I didn't add helpers.py module for simplicity here and have helper function which creates resources directly in this ocs.py example module.